src/MyBrokenClass.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class MyBrokenClass
+{
+    protected string $test;
+
+    public function test(): string
+    {
+        foreach ( $_POST['menu-locations'] as $menu_id ) {
+            echo 'ADD warning severity 10 - non-sanitized input variable';
+        }
+        $functionsCookies = $_COOKIE;
+
+        $functionsPost = $_POST;
+        var_dump($functionsPost);
```

Comment:
:warning: **Warning**: var_dump() found. Debug code should not normally be used in production (*WordPress.PHP.DevelopmentFunctions.error_log_var_dump*).